static/src/stylesheets/module/preferences/_notifications.scss

```diff
@@ -1,20 +1,239 @@
-.notifications-subscribe-link--has-subscriptions {
+$live-notifications-icon-width: 36px;
+$live-notifications-icon-width: 32px;
+$notifications-button-height: 28px;
+
+@mixin notifcations-button-style($foreground: $neutral-2, $background: #ffffff) {
+    .notifications__button {
+        color: $foreground;
+        border-color: $foreground;
+        border-style: solid;
+
+        .inline-icon {
+            fill: $foreground
+        }
+    }
+
+    .notifications__button--subscribed {
+        color: $background;
+        background-color: $foreground;
+        .inline-icon {
+            fill: $background;
+            background-color: $foreground;
+        }
+
+        &:hover {
+            background-color: darken($foreground, 10%);
+            .inline-icon {
+                background-color: darken($foreground, 10%);
+            }
+        }
+    }
+
+    .notifications__button--unsubscribed {
+        color: $foreground;
+        background-color: transparent;
+        .inline-icon {
+            fill: $foreground;
+            background-color: $background;
+        }
+
+        &:hover {
+            background-color: darken($background, 10%);
+            .inline-icon {
+                background-color: darken($background, 10%);
+            }
+
+        }
+    }
+}
+
+@mixin notifications-button-toned($tone, $foreground: $news-main-1, $background: #ffffff) {
+    .tonal--tone-#{$tone} {
+        @include notifcations-button-style($foreground, $background);
+    }
+}
+
+.live-notifications {
+    margin-bottom: $gs-baseline /2;
+    position: relative;
+}
+
+.notifications__button {
+
+    .inline-icon {
+        svg {
+            width: 1.143em;
```

Comment:
We should be using pixel values in our CSS. They are automatically converted to em at compile time